lib/task.ts

```diff
@@ -0,0 +1,8 @@
+import Axios from 'axios'
+
+export function percyHealthCheck() {
+  return Axios
+    .get('http://localhost:5338/percy/healthcheck')
```

Comment:
While I would normally be concerned about the port not being correct, you can't actually configure the port without breaking anyways. I'm hopeful this health check implementation doesn't live longer than the port bug.